angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
+        '@angular-eslint/component-selector': 'error',
+        '@angular-eslint/contextual-decorator': 'error',
+        '@angular-eslint/directive-selector': 'error',
+        '@angular-eslint/no-attribute-decorator': 'error',
+        '@angular-eslint/no-forward-ref': 'error',
+        '@angular-eslint/no-input-prefix': 'error',
+        '@angular-eslint/no-lifecycle-call': 'error',
+        '@angular-eslint/no-pipe-impure': 'error',
+        '@angular-eslint/no-queries-metadata-property': 'error',
+        '@angular-eslint/pipe-prefix': 'error',
```

Comment:
[@angular-eslint/pipe-prefix](http://codelyzer.com/rules/pipe-prefix)

Reply:
This rule requires application specific configuration.

**Proposal**
Leave this rule disabled, but include it in a section for rules to consider in the README.

Reply:
@TrevorKarjanis this is ready to close